source/includes/stackpath/_scripts.md

```diff
@@ -0,0 +1,116 @@
+## Scripts
+
+Deploy and manage Serverless Scripts used to interact with requests made to the site.
```

Comment:
Not sure about the context of `request` to `site` here, but how about 
```
Deploy and manage Serverless Scripts for interacting with the sites
```
But if the original one fits well in the context, we are good to go 👍🏻

Reply:
I got that from the Stackpath documentation https://support.stackpath.com/hc/en-us/articles/360001473706-Learn-About-Serverless-Scripting-Basics (second sentence) so it fits well within the context